app/src/main/java/org/gnucash/android/ui/util/widget/CalculatorEditText.java

```diff
@@ -192,57 +186,7 @@ public CalculatorKeyboard getCalculatorKeyboard(){
     }
 
     /**
-     * Returns the view Id of the keyboard view
```

Comment:
These methods are probably marked in the IDE as unused, but they are there because the documentation recommends having them when creating custom views http://goo.gl/6Sg4f . So I would prefer to leave them in. 

Also the keyboardKeysLayout can be set in XML and thereby manipulated programmatically (even if we don't do so at the moment).